EM: clarify query parameters for GET of requests and assignments by users and apps in catalog-specific roles (#17822)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/entitlementmanagement-list-assignments.md

GET /identityGovernance/entitlementManagement/assignments
```

## Query parameters

This method supports the `$select`, `$filter`, and `$expand` OData query parameters to help customize the response.

> If the caller is on behalf of a delegated user who is assigned only to catalog-specific delegated administrative roles, or the application has been assigned to catalog-specific administrative roles, the request must supply a filter to indicate a specific access package, such as: `$filter=accessPackage/id eq 'a914b616-e04e-476b-aa37-91038f0b165b'`.

Reviewer: ```suggestion
If the user or app is assigned only to catalog-specific administrative roles, the request must supply a filter to indicate a specific access package, such as: `$filter=accessPackage/id eq 'a914b616-e04e-476b-aa37-91038f0b165b'`.
```
